molecule/null-cr-values-test/kiali-cr.yaml

```diff
@@ -140,3 +140,4 @@ spec:
     web_fqdn: null
     web_root: null
     web_schema: null
+    web_history_mode: null
```

Comment:
As per [coding standards](https://github.com/kiali/kiali-operator/blob/master/DEVELOPING.adoc#are-you-altering-a-kiali-server-configuration-setting), place this in alphabetical order within this group of settings - so `web_history_mode` should be placed before `web_root` and after `web_fqdn`.